src/client/driver/driver-link/send-message-with-confirmation.js

```diff
@@ -0,0 +1,43 @@
+import { Promise, eventSandbox, nativeMethods } from '../deps/hammerhead';
+import INTER_DRIVER_MESSAGES from './messages';
+import generateId from '../generate-id';
+
+
+const MIN_RESPONSE_WAITING_TIMEOUT = 2500;
+const RESEND_MESSAGE_INTERVAL      = 1000;
+
+
+export default function sendMessageWithConfirmation (msg, driverWindow, timeout, NotLoadedErrorCtor) {
```

Comment:
The `sendMessageWithConfirmation` function is used by `ChildDriverLink` and `ParentDriverLink`. But they are different classes